packages/react-router5/modules/RouteProvider.js

```diff
@@ -45,7 +51,11 @@ class RouteProvider extends React.PureComponent {
     }
 
     render() {
-        return <Provider value={this.state}>{this.props.children}</Provider>
+        const value = {
+            router: this.props.router,
```

Comment:
Is this intended?

Reply:
I think yes, since we don't use the react state anymore, we need to inject the router to the children, what's the problem ?

Reply:
Nope, just making sure that was intended :)

Reply:
(since I'll document it)